source/_posts/2018-01-08-how-to-quickly-deploy-a-vuejs-app-to-heroku.markdown

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: "How to quickly deploy a VueJS app to Heroku"
+category: javascript
+comments: true
+author:
+  name: Jorge Bejar
+  email: jorge@wyeworks.com
+  twitter_handle: jmbejar
+  github_handle: jmbejar
+  image: /images/team/jorge-bejar.jpg
+  description: Love to explore and write about Ruby, Javascript, full-stack development and technology in general.
+---
+
+Recently, I was investing some time learning VueJS and I found that it is a very interesting framework to play with. In fact, I've been working on a new project prototype during a few days and wanted to show it to some people, so I wanted to publish it anywhere in the Internet.
```

Comment:
publish it anywhere -> publish it somewhere

Reply:
`a few days`, no sería mejor usar `the last few days`?

Reply:
Wouldn't it be better to use `the last few days` instead of `a few days`?

Reply:
during -> for ?

Reply:
I was investing some time -> I've been investing some time?

Reply:
to play (around) with ?